pwiz_tools/Skyline/SkylineNightly/SkylineNightly.csproj

```diff
@@ -127,6 +127,12 @@
     </Compile>
     <EmbeddedResource Include="SkylineNightly.skytr" />
   </ItemGroup>
+  <ItemGroup>

```

Comment:
I don't think we need this dependency anymore, which is good, because SkylineNightly.exe generally does not have this DLL around in real-world use.